lib/active_merchant/billing/gateways/kushki.rb

```diff
@@ -208,6 +211,29 @@ def add_deferred(post, options)
         }
       end
 
+      def add_product_details(post, options)
+        return unless options[:product_details]
+
+        product_items_array = []
+        options[:product_details].each do |item|
+          product_items_obj = {}
+
+          product_items_obj[:id] = item[:id] if item[:id]
+          product_items_obj[:title] = item[:title] if item[:title]
+          product_items_obj[:price] = item[:price].to_i if item[:price]
+          product_items_obj[:sku] = item[:sku] if item[:sku]
+          product_items_obj[:qauntity] = item[:qauntity].to_i if item[:qauntity]
```

Comment:
This should be `quantity` instead of `qauntity`, both here and in the tests

Reply:
thanks for catching this!